atlassian_jwt_auth/key.py

```diff
@@ -88,6 +89,9 @@ def __init__(self, base_url):
     def _get_session(self):
         if HTTPSPublicKeyRetriever._class_session is None:
             session = cachecontrol.CacheControl(requests.Session())
+            session.proxies = requests.utils.get_environ_proxies(
+                self.base_url)
+            session.trust_env = False
```

Comment:
Is there a difference in behavior between `False` and the default of `None` here or are you just being explicit?

Reply:
I am not sure, I thought it would be safer to go with `False` though.